app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -16,6 +16,7 @@ def calculate
         set_line(:NJ1040_LINE_7_SPOUSE, :line_7_spouse_checkbox)
         set_line(:NJ1040_LINE_7, :calculate_line_7)
         set_line(:NJ1040_LINE_8, :calculate_line_8)
+        set_line(:NJ1040_LINE_9, :calculate_line_9)
```

Comment:
[pebble] why are we setting the LINE_9 value if it is never used? It seems like it is never read by either the XML or PDF generation.

Reply:
It is used to calculate line 13 of the NJ1040 below

Reply:
Couldn't you just call the function `calculate_line_9` below? I see the `calculate_x` functions as used internally in the calculator, and `NJ1040_LINE_X` values as things that are used externally. Because Line 9 is internal only, it seems like it doesn't need to be available to other callers.

Reply:
I see the logic in reserving `NJ1040_LINE_X` for values used externally. However, that complicates the testing story for this. If I stop saving these variables to lines, then I need to make calculate_line_8 and calculate_line_9 public methods for testing, which leads to the following structure in the file. Do these calculator helper methods need to be private at all?

```
calculate_line_8

calculate_line_9

private

calculate_line_6

calculate_line_7

calculate_line_13
```